### PR TITLE
VGUI: Add Database access to panel elements

### PR DESCRIPTION
**Pull request by ZenBre4ker**
This adds database access to checkbox, slider and the combobox

### CODE REVIEW

File: gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dcheckboxlabel_ttt2.lua [hunk 1]
```diff
@@ -88,25 +97,77 @@ function PANEL:SetServerConVar(cvar)
...
+
+	self.databaseInfo = databaseInfo
+
+	database.GetValue(name, itemName, key, function(databaseExists, value)
+		if databaseExists then
+			self:SetValue(value, true)
+		end
+	end)
+
+	self:SetDefaultValue(database.GetDefaultValue(name, itemName, key))
```
Comment (saibotk) [inline gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dcheckboxlabel_ttt2.lua:142]: Does this error or return nil if the database does not exist?

**Review by saibotk — CHANGES_REQUESTED** [review]

File: gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dcheckboxlabel_ttt2.lua (hunk 1 above)
Comment (ZenBre4ker) [reply gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dcheckboxlabel_ttt2.lua:142]: It errors.

File: gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dcheckboxlabel_ttt2.lua [hunk 2]
```diff
@@ -63,6 +63,14 @@ function PANEL:Paint(w, h)
 	return true
 end
 
+---
+-- This is only used temporarily to enable old values for Settings, that we internally inverted
```
Comment (TimGoll) [inline gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dcheckboxlabel_ttt2.lua:67]:
```suggestion
-- This is only used temporarily to keep old variables without breaking the style of "no enable to disable" checkboxes
```

**Review by TimGoll — APPROVED** [review]
LGTM

**Review by saibotk — APPROVED** [review]

**Pull request merged**